helga_jeopardy.py

```diff
@@ -125,7 +150,7 @@ def eval_potential_answer(input_line, answer):
 
     return correct, partial, ratio
 
-def reveal_answer(client, channel, question_id, answer, mongo_db=db.jeopardy):
+def reveal_answer(client, channel, question_id, answer, random=True, mongo_db=db.jeopardy):
```

Comment:
When adding parameters to an existing function, it is generally preferable to add the new parameters at the end. That way, you don't break any existing callers who are using positional arguments and relying on the order of those arguments to be what they expect.

For example, any usage of `reveal_answer` like this would break:

```python
reveal_answer(client, channel, question_id, answer, db.trivia)
```

```suggestion
def reveal_answer(client, channel, question_id, answer, mongo_db=db.jeopardy, random=True):
```